<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>0</height>
        </size>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>995</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenPolicyDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionOpenUserRegistrySource"/>
    <addaction name="actionOpenMachineRegistrySource"/>
    <addaction name="actionSaveRegistrySource"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Action</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionAddRemoveColumns"/>
    <addaction name="separator"/>
    <addaction name="actionSmallIconView"/>
    <addaction name="actionLargeIconView"/>
    <addaction name="actionCompactListView"/>
    <addaction name="actionDetailedListView"/>
    <addaction name="separator"/>
    <addaction name="actionSelectDC"/>
    <addaction name="separator"/>
    <addaction name="actionCustomize"/>
    <addaction name="actionEditFilter"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGoBack"/>
   <addaction name="actionGoForward"/>
   <addaction name="actionGoUp"/>
   <addaction name="actionReload"/>
   <addaction name="separator"/>
   <addaction name="actionSmallIconView"/>
   <addaction name="actionLargeIconView"/>
   <addaction name="actionCompactListView"/>
   <addaction name="actionDetailedListView"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionGoBack">
   <property name="icon">
    <iconset theme="go-previous">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Go &amp;Back</string>
   </property>
   <property name="toolTip">
    <string>Go Back</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionGoForward">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Go &amp;Forward</string>
   </property>
   <property name="toolTip">
    <string>Go Forward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionGoUp">
   <property name="icon">
    <iconset theme="go-up">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Go &amp;Up</string>
   </property>
   <property name="toolTip">
    <string>Go Up</string>
   </property>
   <property name="shortcut">
    <string>Alt+Up</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset theme="reload">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSmallIconView">
   <property name="text">
    <string>&amp;Small Icon View</string>
   </property>
   <property name="toolTip">
    <string>Small Icon View</string>
   </property>
  </action>
  <action name="actionLargeIconView">
   <property name="text">
    <string>&amp;Large Icon View</string>
   </property>
   <property name="toolTip">
    <string>Large Icon View</string>
   </property>
  </action>
  <action name="actionCompactListView">
   <property name="text">
    <string>&amp;Compact List View</string>
   </property>
   <property name="toolTip">
    <string>Compact List View</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset theme="settings">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
   <property name="toolTip">
    <string>Options</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAddRemoveColumns">
   <property name="text">
    <string>&amp;Add/Remove Columns</string>
   </property>
   <property name="toolTip">
    <string>Add/Remove Columns</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDetailedListView">
   <property name="text">
    <string>&amp;Detailed List View</string>
   </property>
   <property name="toolTip">
    <string>Detailed List View</string>
   </property>
  </action>
  <action name="actionSelectDC">
   <property name="text">
    <string>&amp;Select DC</string>
   </property>
   <property name="toolTip">
    <string>Select Domain Controller</string>
   </property>
  </action>
  <action name="actionCustomize">
   <property name="text">
    <string>&amp;Customize</string>
   </property>
   <property name="toolTip">
    <string>Customize</string>
   </property>
  </action>
  <action name="actionEditFilter">
   <property name="text">
    <string>&amp;Edit filter</string>
   </property>
   <property name="toolTip">
    <string>Edit filter</string>
   </property>
  </action>
  <action name="actionOpenPolicyDirectory">
   <property name="text">
    <string>Open &amp;Policy Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenUserRegistrySource">
   <property name="text">
    <string>Open &amp;User Registry Source</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSaveRegistrySource">
   <property name="text">
    <string>&amp;Save Registry Source</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenMachineRegistrySource">
   <property name="text">
    <string>Open &amp;Machine Registry Source</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
